### PR TITLE
open in colab badge

### PR DESCRIPTION
**Pull request by vainaijr**

### CODE REVIEW

**Comment by vainaijr** [comment]
for some reason it does not appear on my github, I edited only one badge, but it does not show on my github.

**Comment by MisaOgura** [comment]
Hi @vainaijr,

No worries, the notebook preview on GitHub doesn't often load.

As you can see below, for both `activation_maximization_colab.ipynb` and `activation_maximization.ipynb`, there are 2 badges:

Screen Shot 2019-09-19 at 15 00 42
Screen Shot 2019-09-19 at 15 00 48

My preference is to keep one which links to the notebook from this repo. i.e. `https://colab.research.google.com/github/MisaOgura/flashtorch/blob/master/examples/activation_maximization_colab.ipynb`.

Many thanks,
Misa

**Comment by vainaijr** [comment]
I added only one badge, but it shows two,

Screenshot (455)
Screenshot (456)

**Comment by MisaOgura** [comment]
Hi @vainaijr,

Great, I can see that now the notebooks have one button.

One final request - can you change both links to `https://colab.research.google.com/github/MisaOgura/flashtorch/blob/master/examples/activation_maximization_colab.ipynb`, as the both links should redirect users to the colab version of the notebook on the master branch of this repository.

Many thanks,
Misa

**Comment by vainaijr** [comment]
I had to edit through jupyter notebook, I think it will work now.

File: examples/activation_maximization_colab.ipynb [hunk 1]
```diff
@@ -1,303 +1,405 @@
...
-}
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/vainaijr/flashtorch/blob/master/examples/activation_maximization_colab.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (MisaOgura) [inline examples/activation_maximization_colab.ipynb]: Hi @vainaijr,

Please change the link to point to `https://colab.research.google.com/github/MisaOgura/flashtorch/blob/master/examples/activation_maximization_colab.ipynb` as discussed.

Many thanks,
Misa

Comment (vainaijr) [reply examples/activation_maximization_colab.ipynb]: the default colab generated notebook, adds a open in colab button linking to file in my repo, so I changed both files from jupyter notebook., I added link to activation_maximization_colab in both

**Comment by MisaOgura** [comment]
All good @vainaijr, thanks for your contribution.

**Pull request merged**